app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -97,6 +98,15 @@ class ChatActivity : AppCompatActivity(), IChatView {
         chatPresenter.setUp()
         chatPresenter.checkPreferences()
         setEditText()
+        /*
+         Adding on swipe listener for swipe between @see(SkillListingFragment.kt) and ChatActivity.kt
+         */
+        rv_chat_feed.setOnTouchListener(object : OnSwipeListener(this) {
+            override fun onSwipeRight() {
+                val i = Intent(context, SkillsActivity::class.java)
+                startActivity(i)
```

Comment:
i --> intent.
@codedsun It would be really nice if you could please see that you check for all the files when somebody reviews a PR and requests a change. Sometimes there are many places where you have done the same thing, it's not possible to mark each and every line. It unnecessary delays the PRs.

Reply:
@arundhati24 What I saw that in same file

```
fun openSettings(view: View) {
        val i = Intent(this, SkillsActivity::class.java)
        startActivity(i)
        finish()
    }
``` 

we are having names like this, so I defined the same just to follow the coding style.